docs/internals/contributing/writing-code/coding-style.txt

```diff
@@ -52,6 +52,41 @@ Python style
   single quote. Don't waste time doing unrelated refactoring of existing code
   to conform to this style.
 
+* String variable interpolation may use `%-formatting`__, `f-strings`__, or
+  :py:meth:`str.format` as appropriate, with the goal of maximizing code
+  readability.
```

Comment:
You can use `:py:ref:`:
```suggestion
* String variable interpolation may use
  :py:ref:`%-formatting <old-string-formatting>`,
  :py:ref:`f-strings <f-strings>`, or :py:meth:`str.format` as appropriate,
  with the goal of maximizing code readability.
```

Reply:
Super. If only Sphinx could make coffee. 😀